docs/LibcxxSupport.md

```diff
@@ -1,63 +1,194 @@
-codecvt | Yes | - |
-compare | No | - |
-complex | Yes | - |
-condition_variable | Yes | - |
+concepts | No | Header is not provided, C++20 is not yet supported. |
+
+## Coroutines Library
+Header | Supported | Comments |
+:---:|:---:|:---|
+coroutines | No | Header is not provided, C++20 is not yet supported. |
+
+## Utilities Library
+Header | Supported | Comments |
+:---:|:---:|:---|
+any | No | C++17 is not yet supported. |
+bitset | Yes | - |
+compare | No | C++20 is not yet supported. |
 csetjmp | Yes | - |
-csignal | Yes | - |
-cstdarg | No | - |
+csignal | No | - |
+cstdarg | Yes | - |
 cstddef | Yes | - |
-cstdint | Yes | - |
-cstdio | Partial | All I/O functions implicitly call out to untrusted host. <br> **Unsupported functions:** ferror(), vscanf() |
 cstdlib | Partial | **Unsupported functions:** at_quick_exit(), quick_exit() |
-cstring | Partial | **Unsupported functions:** strcpy(), strcat(), strncat(), strchr(), strcspn(), strspn() |
-ctime | Yes | All time functions implicitly call out to untrusted host for time values. The resulting time values should not be used for security purposes. <br> **Supported functions:** time(), gettimeofday(), clock_gettime(), nanosleep(). _Please note that clock_gettime() only supports CLOCK_REALTIME_ |
-cwchar | Partial | **Unsupported functions:** wscanf(), wprintf() |
-cwctype | Yes | - |
-cuchar | No | - |
-exception | Partial | **Unsupported functions:** throw_with_nested(), rethrow_if_nested() |
+ctime | Partial | All time functions implicitly call out to untrusted host for time values. The resulting time values should not be used for security purposes. <br> **Supported functions:** time(), gettimeofday(), clock_gettime(), nanosleep(). _Please note that clock_gettime() only supports CLOCK_REALTIME_ |
+chrono | Partial | Supported as part of C++11. All time functions implicitly call out to untrusted host for time values. The resulting time values should not be used for security purposes. <br> **Supported classes:** system_clock, treat_as_floating_point, duration_values |
 functional | No | - |
-future | Yes | - |
-fstream | Yes | All I/O functions implicitly call out to untrusted host. |
-initializer_list | Yes | - |
-iomanip | Partial | **Unsupported functions:** get_money(), put_money(), get_time(), put_time() |
-ios | Partial | **Unsupported functions:** nounitbuf(), nouppercase(), noshowpos(), noshowpoint(), noshowbase(), noboolalpha() |
-istream | Yes | - |
-iterator | Partial | **Unsupported functions:** make_reverse_iterator(), make_move_iterator(), front_inserter(), back_inserter(), inserter(), begin(), cbegin(), rbegin(), crbegin() |
+initializer_list | Yes | Supported as part of C++11. |
+optional | No | C++17 is not yet supported. |
+tuple | Partial | Supported as part of C++11, known issues with apply() template. |
+type_traits | Yes | Supported as part of C++11. |
+typeindex | Yes | Supported as part of C++11. |
+typeinfo | No | - |
```

Comment:
typeinfo ought to be supported since RTTI and exception handling depend on it.

Reply:
Changed to supported.